blockchainServices/liqudate.js

```diff
@@ -77,11 +81,30 @@ const checkTradeIsClose = async (tradeId, contract) => {
                 block: "0",
                 exchangeAddress: contract.address
             }
-            post(obj, "remove")
-            logger.log('info',  {message: `tradeId: ${tradeId} closed`,})
+        post(obj, "remove")
+        logger.log('info',  {message: `tradeId: ${tradeId} closed`,})
+        
+        sendMessage(tradeId.toString());
```

Comment:
wrap in if telegram bot enabled, add telegram bot enabled to config, set to true as default

Reply:
also this should not be here this should be in the liquidate trade function

Reply:
also we should check successful tx first so a provider. wait for tx maybe necessary but it should not be blocking